### ISSUE TITLE
Dependency issue on some clients

### ISSUE THREAD

**Issue by tenplus1** [Andrey2470T/multidecor#11]
Some clients are getting the following error:

```
ModError: Failed to load and run script from C:\minetest\bin\..\mods\multidecor\craft_ingredients\init.lua:
...netest\bin\..\mods\multidecor\craft_ingredients\init.lua:20: attempt to index global 'multidecor' (a nil value)
stack traceback:
...netest\bin\..\mods\multidecor\craft_ingredients\init.lua:20: in main chunk

```
By adding decor_api to the depends line in the mod.conf files of the other mods, it makes sure it will load first stopping the error.

**Comment by Andrey2470T** [comment]
Thanks for the issue reporting. That will be fixed in 1.3.4.

**Comment by Andrey2470T** [comment]
Fixed in the new release.
